### ISSUE TITLE
[4.0] Spinner icon displays out of bound

### ISSUE THREAD

**Issue by Quy** [joomla/joomla-cms#32948]
### Steps to reproduce the issue
Go to System > Language Overrides.
Select a Language & Client.
Click New button.
Click cursor in Search Text field.

### Actual result

![language-override](https://user-images.githubusercontent.com/368084/113182425-bc821480-9207-11eb-861c-f10a6a8b6476.gif)

**Comment by ghost** [comment]
Issue confirmed.

**Comment by RahulJha393** [comment]
I would like to fix this issue . Kindly guide where I can start from

**Comment by YatharthVyas** [comment]
I was able to reproduce it but not consistently. I figured out that the issue has something to do with the span#refresh-status in /administrator/components/com_languages/tmpl/override/edit.php
I'd like to contribute to this. Could you please assign it to me

**Comment by ghost** [comment]
@YatharthVyas you can make a pull request without an issue is assigned to you.

**Comment by YatharthVyas** [comment]
I figured out and solved the issue in my local version. I solved this by making the following change:

Original
`#refresh-status.show {
	display: grid;
}`

Modified
`#refresh-status.show {
	display: block;
}`

I will be adding a more detailed explanation behind why this caused the bug in my PR.

This change was made in my /media folder which is added to .**gitignore** because I believe it's contents are installed using `npm ci` and `composer install` commands. 

So, where exactly do I put my modified css files so that it reflects in the PR? 

The files modified are:
media/com_languages/css/overrider.min.css
media/com_languages/css/overrider.css
build/media_source/com_languages/css/overrider.css
